index.html

```diff
@@ -51,33 +58,185 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
     </section>
 
     <section class="middle-section">
```

Comment:
Please consider using `display: grid;` instead of `display: flex;` for the container to align the elements as it's a project requirement.